### DISCUSSION TITLE
What's the point of RepeatDataset?

### DISCUSSION THREAD

**Discussion by Divelix** [open-mmlab/mmdetection3d#2286] [Q&A] [answered]
As I understand, doubling `max_epochs` in `EpochBasedRunner` equals to setting `times=2` in `RepeatDataset`. So what's the point of RepeatDataset then?

**Comment by JingweiZhang12** [comment, marked as answer]
@Divelix The learning scheduler is different in doubling  max_epoch and setting times=2 in RepeatDataset.

**Reply by Divelix** [reply]
Not quite clear how exactly is it different? Lets say I set RepeatDataset=2: can I apply different augmentations for 1 and 2?

**Reply by JingweiZhang12** [reply]
You can't. RepeatDataset with times=2 only repeats the dataset twice. It has no related to augmentation.
